django/core/management/commands/test.py

```diff
@@ -47,6 +47,10 @@ def add_arguments(self, parser):
             help='Overrides the default address where the live server (used '
                  'with LiveServerTestCase) is expected to run from. The '
                  'default value is localhost:8081-8179.')
+        parser.add_argument('--tag', action='append', dest='tags',
```

Comment:
@carljm -- should these options be added here or in [DiscoverRunner](https://github.com/django/django/blob/fcd08c175787e909b3eb98f756317a07741c48dd/django/test/runner.py#L376-L396)? This looks backwards incompatible as is because it forces all runners to accept tags/exclude_tags in `test_runner.run_tests()`?